packages/graphql-language-service-server/src/__tests__/MessageProcessor-test.ts

```diff
@@ -208,56 +208,120 @@ describe('MessageProcessor', () => {
     await expect(result[0].uri).toEqual(`file://${queryDir}/test3.graphql`);
   });
 
-  it('getQueryAndRange finds queries in tagged templates', async () => {
-    const text = `
-// @flow
-import {gql} from 'react-apollo';
-import type {B} from 'B';
-import A from './A';
-
-const QUERY = gql\`
-query Test {
-  test {
-    value
-    ...FragmentsComment
-  }
-}
-\${A.fragments.test}
-\`
-
-export function Example(arg: string) {}`;
-
-    const contents = getQueryAndRange(text, 'test.js');
-    expect(contents[0].query).toEqual(`
-query Test {
-  test {
-    value
-    ...FragmentsComment
-  }
-}
-`);
-  });
+  describe('getQueryRange', () => {
+    describe('js/ts', () => {
```

Comment:
so this test is marked as a js/ts test, but i see a flow js file? can we have a test for a flow .js, normal es6 .js, and a .ts file? just to be certain everything is hunky dory

Reply:
Yeah, I'll add tests for more extensions :+1: 
Since it's regexp based now we don't really need to care about any other content than the actual graphql tag, which means it's only really the extension that's interesting to test, at least in my view.